### ISSUE TITLE
CMAKE: Configuring incomplete, errors occurred!

### ISSUE THREAD

**Issue by Aryangh1379** [mmatyas/supermariowar#302]
Hi, I get errors when compiling with cmake... help please. https://paste.rs/ztHOS.txt
distro: arch
i'm using this version: https://github.com/mmatyas/supermariowar/archive/refs/tags/2023w03.tar.gz
simple `cmake ..` command also fails: https://paste.rs/plytd.txt
thanks for your awsome work.

**Comment by mmatyas** [comment]
Hi! CMake tells the issue:
```
CMake Error at CMakeLists.txt:144 (message):
  It seems you've forgot to clone the dependencies; dependencies/enet is not
  a git repository.Please run `git submodule update --init` first.
```
The archives autogenerated by GitHub do not include all the files, but you can download the repository using any Git client. Also note that the tags don't have much significance here, so you might prefer to use the latest version.

**Comment by Aryangh1379** [comment]
really appreciate your quick answer. I did that, the command returned no output meaning successfully ran. same error when making yet: https://paste.rs/M96Xe.txt
anyways, I decided to use https://github.com/mmatyas/supermariowar/releases/tag/continuous and use the binaries. works fine also I packaged it for archlinux: https://aur.archlinux.org/packages/supermariowar-bin
hoping people use it. great luck!
